pkg/resources/management.cattle.io/v3/clusterroletemplatebinding/validator.go

```diff
@@ -73,41 +73,31 @@ func (a *admitter) Admit(request *admission.Request) (*admissionv1.AdmissionResp
 	listTrace := trace.New("clusterRoleTemplateBindingValidator Admit", trace.Field{Key: "user", Value: request.UserInfo.Username})
 	defer listTrace.LogIfLong(admission.SlowTraceDuration)
 
+	fieldPath := field.NewPath("clusterroletemplatebinding")
+	var fieldErr *field.Error
```

Comment:
I think you could slightly re-arrange the error variables for better readability, as suggested in this comment in the PRTB PR.
https://github.com/rancher/webhook/pull/228#discussion_r1261776123

Reply:
I believe I've changed it to what you asked. LMK